include/spirv-tools/optimizer.hpp

```diff
@@ -838,6 +838,14 @@ Optimizer::PassToken CreateWrapOpKillPass();
 // capabilities.
 Optimizer::PassToken CreateAmdExtToKhrPass();
 
+// Replaces the internal version of GLSLstd450 OpInterpolateAt* instructions
+// with the externally valid version. The internal version allows an OpLoad
+// of the interpolant for the first argument. This pass removes the OpLoad and
+// replace with itspointer. glslang creates the internal version for HLSL. This
+// pass will be called at the end of glslang HLSL legalization after
+// interpolants have been propagated into their final positions.
+Optimizer::PassToken CreateInterpFixupPass();
```

Comment:
I would prefer if we not use abbreviations that are not well known.  It can be difficult to read.  Please use `CreateInterpolationFixupPass`, or something like that.

Reply:
OK